pandas/tests/indexing/test_indexing.py

```diff
@@ -710,6 +724,27 @@ def test_float_index_at_iat(self):
         for i in range(len(s)):
             assert s.iat[i] == i + 1
 
+    def test_mixed_index_at_iat(self):
+        s = Series([1, 2, 3, 4, 5], index=['a', 'b', 'c', 1, 2])
+        for el, item in s.iteritems():
+            assert s.at[el] == item
+        for i in range(len(s)):
```

Comment:
can you check that we have similar tests for .loc (I think so), if not can you add a loc version (inside same test is ok). maybe even just add the .loc tests and move from elsewhere.

you can parameterize these over .at / .loc